src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,27 @@
 package mate.academy.service;
 
-public class AuthenticationService {
+import mate.academy.model.User;
+
+public class AuthenticationService extends UserService {
```

Comment:
this class can't be inherited from UserService. in this case, you should use aggregation